doc/api/crypto.md

```diff
@@ -1731,6 +1731,14 @@ buffer containing the data encoded as DER. Note that Node.js itself does not
 accept DER, it is supported for interoperability with other libraries such as
 WebCrypto only.
 
+Private keys can be encrypted if the `type` is PKCS#8 or the `format` is PEM.
+If a `cipher` is specified and PKCS#8 was selected, an `EncryptedPrivateKeyInfo`
```

Comment:
But what is `EncryptedPrivateKeyInfo` ? A quick google search turns up a bunch of Java API references...

Reply:
What about "an `EncryptedPrivateKeyInfo` structure as defined in section 6 of RFC 5208" (with a link to the RFC)?

Reply:
I think the important thing here in our docs is to say that the key will be encrypted. That the ASN.1 structure used is EncryptedPrivateKeyInfo doesn't seem really relevant at this level - and since that's the only ASN.1 object that is encrypted in PKCS#8, they won't have any trouble looking it up if they need to. Linking `PKCS#8` to the RFC (or the PKCS standard) is a good idea, IMO.